### PR TITLE
Add all open-power-host-os packages to automatic update list

### PR DESCRIPTION
**Pull request by ghost**
All package repositories under the organization should be kept up-to-date.

### CODE REVIEW

**Comment by ghost** [comment]
This depends on open-power-host-os/versions#62

File: lib/package.py [hunk 1]
```diff
@@ -89,19 +89,20 @@ def __repr__(self):
         return self.name
 
 
-    def download_files(self):
+    def download_files(self, recurse=True):
```
Comment (maurorodrigues) [inline lib/package.py:92]: I think you meant recursive not recurse (which in pr-br is 'recurso' )

Comment (maurorodrigues) [reply lib/package.py:92]: other than that +1..

Comment (ghost) [reply lib/package.py:92]: The verb is form is ["recurse"](https://en.wiktionary.org/wiki/recurse). I can change to "recursive" or "recursively" if you prefer.

Comment (maurorodrigues) [reply lib/package.py:92]: No need, if you're right you're right. I swear I translated it before being pedantic =).. +1

**Review by maurorodrigues — APPROVED** [review]

**Pull request merged**